oaknational/Oak-Web-Application
# feat: update search mv version [LESQ-1023]

## Description

Music year: {owa_music_year}

- update search page MV to latest version

## Issue(s)

Fixes #
- search MV not refreshing
- all-ks returned in filter on search page list (not currently happening but will when all-ks is published to production)

## How to test

1. Go to https://deploy-preview-2702--oak-web-application.netlify.thenational.academy/teachers/search?term=mcbeth
3. You should see the subject list is reduced
4. You should not see any other changes

## Screenshots

How it used to look (delete if n/a):
![Screenshot 2024-08-15 at 11 41 46](https://github.com/user-attachments/assets/5b37e1a6-ef05-489f-99f0-f64a8a0d64de)


How it should now look:
![Screenshot 2024-08-15 at 11 41 33](https://github.com/user-attachments/assets/1d4ef927-5916-4b49-9736-4a2f99bd3f4e)
